blocks/core/ff_container/ff_container-filter-dropdown/ff_container-filter-dropdown.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+$(function(){ 
+	$("[data-ff-action='filter-toggler']").click(function(){
+		var filterTogglerText = $("[data-ff='filter-content']").is(':visible') ? 'Filter this list' : 'Hide filters' ;
```

Comment:
It might be a good idea to make the visibility state a bit more explicit, in order to prevent the toggles getting out or sync; something like (pseudo code)
```
var isVisible = content:is(:visible); 
var newText = isVisible ? 'Filter this list' : 'Hide filters';
if (isVisible) {
$(content).hide();
$(icon).removeClass(visibleClass).addClass(hiddenClass);
else {
$(content).show();
$(icon.removeClass(hiddenClass).addClass(visibleClass);
}
$(text).text(newText);
```